src/App.tsx

```diff
@@ -15,36 +16,105 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App: React.FC = () => {
+enum SortType {
+  Alphabet = 'name',
+  Length = 'length',
+}
+
+interface PreparedGoodsInterface {
+  field: SortType | '';
+  isReverse: boolean;
+}
+
+const prepareGoods = (
+  goods: string[],
+  { field, isReverse }: PreparedGoodsInterface,
+) => {
+  const preparedGoods = [...goods];
+
+  if (field) {
+    preparedGoods.sort((good1, good2) => {
+      switch (field) {
+        case SortType.Alphabet: {
+          return good1.localeCompare(good2);
+        }
+
+        case SortType.Length: {
+          return good1.length - good2.length;
+        }
+
+        default:
+          return 0;
+      }
+    });
+  }
+
+  return isReverse ? preparedGoods.reverse() : preparedGoods;
+};
+
+export const App = () => {
+  const [sortField, setSortField] = useState<SortType | ''>(SortType.Alphabet);
```

Comment:
Consider initializing `sortField` with an empty string ('') instead of `SortType.Alphabet` to match the `PreparedGoodsInterface` type definition, which allows for an empty string. This will ensure consistency and prevent potential type errors.